src/AutoRest.CSharp/LowLevel/Output/Samples/DpgOperationSample.cs

```diff
@@ -142,7 +142,7 @@ public IEnumerable<ValueExpression> GetValueExpressionsForParameters(IEnumerable
                     }
                     // if it is optional, we just do not put it in the map indicates that in the invocation we could omit it
                 }
-                else
+                else if (!result.ContainsKey(parameter.Name))
```

Comment:
under what case the parameter will be duplicated?

Reply:
Is this fixing this issue? https://github.com/Azure/autorest.csharp/issues/4151

Reply:
Yes, but this happens before I did the current change. I'll revert this until next time we see this.